components/automate-ui/src/app/entities/api-tokens/api-token.reducer.ts

```diff
@@ -45,7 +45,12 @@ export function apiTokenEntityReducer(
   switch (action.type) {
 
     case ApiTokenActionTypes.GET_ALL: {
-      return set(STATUS, EntityStatus.loading, state) as ApiTokenEntityState;
+      return set(
```

Comment:
sorry, ignore all this and everything not on user-management or user-details 😅 this is branched off https://github.com/chef/automate/pull/2130

i've commented on the relevant changes in this PR